variables.tf

```diff
@@ -10,6 +10,12 @@ locals {
   repo_name      = "terraform-aws-asg-rolling-restart-lambda"
 }
 
+variable "lambda_version_constraint" {
+  default     = "~1.0.1-rc9"
```

Comment:
should this be externalized to call attention to it?

Reply:
By that you mean remove the default value?

Reply:
IMO if we remove the default value, this becomes a major release as there's a new required parameter, and I think accepting the default for the version of the module you're using is usually acceptable.

Reply:
I meant just moving it to a tfvars file. It seems weird to have embedded in variables.tf.. but then again since it's a module, I guess that doesn't make sense either so this is probably best the way you have it.

Reply:
Ah. This is where the car is defined, so it _is_ externalized. The parent module can optionally override this default.